src/raster/tests.rs

```diff
@@ -5,42 +5,26 @@ use crate::raster::{
     ByteBuffer, ColorEntry, ColorInterpretation, ColorTable, GdalDataType, RasterCreationOption,
     StatisticsAll, StatisticsMinMax,
 };
-use crate::test_utils::TempFixture;
+use crate::test_utils::{fixture, TempFixture};
 use crate::vsi::unlink_mem_file;
 use crate::DriverManager;
 use std::path::Path;
 
 #[cfg(feature = "ndarray")]
 use ndarray::arr2;
 
-macro_rules! fixture {
```

Comment:
I'm a bit confused here, but :heart_eyes:

Reply:
@lnicola It should have been removed in #342, but somehow was missed.